index.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
```

Comment:
Both the Discovery and CDS Service endpoints return a `Content-Type` of `text/html`. We should be returning `application/json; charset=utf-8`.